moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -34,22 +34,30 @@
 
 /* Author: E. Gil Jones */
 
-#include <ros/ros.h>
+#include <chomp_motion_planner/chomp_optimizer.h>
 #include <chomp_motion_planner/chomp_planner.h>
 #include <chomp_motion_planner/chomp_trajectory.h>
-#include <chomp_motion_planner/chomp_optimizer.h>
 #include <moveit/robot_state/conversions.h>
+#include <tf2/LinearMath/Quaternion.h>
+#include <tf2_eigen/tf2_eigen.h>
```

Comment:
These need a header guard, unfortunately (https://github.com/ros-planning/moveit2/runs/4263445115?check_suite_focus=true#step:8:361), since Rolling the geometry2 headers all switched to `.hpp`. See https://github.com/ros-planning/moveit2/blob/main/moveit_core/robot_state/src/robot_state.cpp#L43 for an example.

Reply:
Fixed.